global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.119",
+    "version": "9.0.305",
```

Comment:
Any reason why you want to update to .NET 9 ?
It requires a tad bit more changes to migrate .NET version (hence the CI failing)
I was planning on upgrading to .NET 10 next month because it's LTS (we are on 8, which is also LTS)

Reply:
I'm not sure why I changed that, I had some issues building it locally so that might've been part of that... 
If you were to upgrade to 10 later, just do that then. No need to do work you'll undo later.

Reply:
Ok :+1: just need to remove this change then